x-pack/plugins/reporting/server/usage/__snapshots__/reporting_usage_collector.test.ts.snap

```diff
@@ -2120,6 +2394,288 @@ Object {
+      },
+      "total": 0,
+    },
+    "_all": 0,
+    "csv_searchsource": Object {
+      "app": Object {
+        "canvas workpad": 0,
+        "dashboard": 0,
+        "search": 0,
+        "visualization": 0,
+      },
+      "available": true,
+      "deprecated": 0,
+      "layout": Object {
+        "canvas": 0,
+        "preserve_layout": 0,
+        "print": 0,
+      },
+      "total": 0,
+    },
+    "csv_searchsource_immediate": Object {
+      "app": Object {
+        "canvas workpad": 0,
+        "dashboard": 0,
+        "search": 0,
+        "visualization": 0,
+      },
+      "available": true,
+      "deprecated": 0,
+      "layout": Object {
+        "canvas": 0,
+        "preserve_layout": 0,
+        "print": 0,
+      },
+      "total": 0,
+    },
+    "output_size": undefined,
+    "printable_pdf": Object {
+      "app": Object {
+        "canvas workpad": 0,
+        "dashboard": 0,
+        "search": 0,
+        "visualization": 0,
+      },
+      "available": true,
+      "deprecated": 0,
+      "layout": Object {
+        "canvas": 0,
+        "preserve_layout": 0,
+        "print": 0,
+      },
+      "total": 0,
+    },
+    "printable_pdf_v2": Object {
+      "app": Object {
+        "canvas workpad": 0,
+        "dashboard": 0,
+        "search": 0,
+        "visualization": 0,
+      },
+      "available": true,
+      "deprecated": 0,
+      "layout": Object {
+        "canvas": 0,
+        "preserve_layout": 0,
+        "print": 0,
+      },
+      "total": 0,
+    },
+    "status": Object {
+      "completed": 0,
+      "failed": 0,
+    },
+    "statuses": undefined,
+  },
+  "output_size": Object {
+    "1.0": 1156282,
+    "25.0": 1156282,
+    "5.0": 1156282,
+    "50.0": 1158078.5,
+    "75.0": 1159875,
+    "95.0": 1159875,
+    "99.0": 1159875,
+  },
+  "printable_pdf": Object {
+    "app": Object {
+      "canvas workpad": 0,
+      "dashboard": 0,
+      "search": 0,
+      "visualization": 0,
+    },
+    "available": true,
+    "deprecated": 0,
+    "layout": Object {
+      "canvas": 0,
+      "preserve_layout": 0,
+      "print": 0,
+    },
+    "total": 0,
+  },
+  "printable_pdf_v2": Object {
+    "app": Object {
+      "canvas workpad": 0,
+      "dashboard": 0,
+      "search": 0,
+      "visualization": 0,
+    },
+    "available": true,
+    "deprecated": 0,
+    "layout": Object {
+      "canvas": 0,
+      "preserve_layout": 0,
+      "print": 0,
+    },
+    "metrics": Object {
+      "csv_rows": Object {
+        "values": Object {
+          "50.0": null,
+          "75.0": null,
+          "95.0": null,
+          "99.0": null,
+        },
+      },
+      "pdf_cpu": Object {
+        "values": Object {
+          "50.0": 0,
+          "75.0": 0,
+          "95.0": 0,
+          "99.0": 0,
+        },
+      },
+      "pdf_memory": Object {
+        "values": Object {
+          "50.0": 164.175,
+          "75.0": 165.3,
+          "95.0": 165.3,
+          "99.0": 165.3,
```

Comment:
Just FYI: These numbers have decimals, but the schema declared them as `long`. Are we OK with dropping the decimals when indexing them in the remote end?

Reply:
That's interesting. The same will be the case with our `sizes/output_size` percentiles. I suppose it's ok but it will make code and tests a bit unclear based on what we see in the remote. I guess we should not consider the code a guide for what we see on the remote side. But if we need to down the road, we can build an adapter for the internal fields vs their remote counterparts.